package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "nullshield",
-  "version": "0.4.0",
+  "version": "0.5.0",
   "description": "A package containing Option and Result types",
   "main": "index.js",
   "types": "index.d.ts",
   "scripts": {
-    "test": "nyc mocha",
+    "test": "tsc && nyc mocha",
```

Comment:
Did you mean to leave in `tsc &&` here?  I never found it necessary to run the compile step before testing, but maybe things have changed?

Reply:
yes, because you can have typescript errors but have the tests still running ok.  This will point out those errors.

Reply:
cool deal